psi4/src/export_wavefunction.cc

```diff
@@ -245,7 +250,8 @@ void export_wavefunction(py::module& m) {
         .def("set_PCM", &Wavefunction::set_PCM, "Set the PCM object")
         .def("get_PCM", &Wavefunction::get_PCM, "Get the PCM object")
 #endif
-        .def("PCM_enabled", &Wavefunction::PCM_enabled, "Whether running a PCM calculation");
+        .def("PCM_enabled", &Wavefunction::PCM_enabled, "Whether running a PCM calculation")
+        .def_readwrite("external_cpscf_perturbations", &Wavefunction::external_cpscf_perturbations_, "Map with functions to compute external perturbations for CPSCF");
```

Comment:
> Can the perturbed contributions from external potentials be added as data members to the HF class?

Definitely. This is the better place for them to live 👍 Do you have a good name in mind? Otherwise I'll transfer it 'as is' and we can bikeshed the name later on.

> Also, it'd be preferable for the new data member (wherever it ends up living)

Sure, that's how it should be.

>  It should be possible to bind it as readwrite property

Can be bound via `def_property` (i.e., `&set_perturbations`, `&get_perturbations` of the respective class).

Reply:
Done. I decided against `def_property` and instead to go in line with the `push_back` stuff that is used for the ground state.